pages/common/pg_restore.md

```diff
@@ -0,0 +1,23 @@
+# pg_restore
+
+> Restore a PostgreSQL database from an archive file created by pg_dump
+
+- restore an archive into an existed database
+
+`pg_restore -d {{db_name}} {{archive_file.dump}}`
+
+- customize host, port, username
+
+`pg_restore -h {{localhost}} -p {{5432}} -U {{username}} -d {{db_name}} {{archive_file.dump}}`
+
+- not require ownership of objects to match the original database
```

Comment:
i don't think `--no-owner` option should be in tldr. It is not a trivial thing. Not for newbies. I would rather split the previous example to several examples for each option.